### PR TITLE
Utilities - Roll Function

### PR DESCRIPTION
**Pull request by ragecryx**
Created `Roll(expression)` function that resolves expressions like `3d6+2` etc.

### CODE REVIEW

File: src/lib/utils.js [hunk 1]
```diff
@@ -1,4 +1,61 @@
...
+import Chance from "chance";
+
+const DEFAULT_SEED = process.env.NODE_ENV === "test" ? "FIXED_SEED" : undefined;
+
+const DiceRandom = new Chance(DEFAULT_SEED);
+
+
+const DiceExpression = /^\d*d\d{1,20}$/img;
+const SingleDieExpression = /^d\d{1,20}$/img;
+const BasicMathDelimeters = /([\+\-\*\/])/g;
```
Comment (dimgrav) [inline src/lib/utils.js]: *Delimiters :P

Comment (ragecryx) [reply src/lib/utils.js]: Ooops. Fixing incoming.

File: src/lib/utils.js [hunk 2]
```diff
@@ -1,4 +1,61 @@
...
+
+    // Evaluate the fixed expression that remains (with rolls resolved)
+    // TODO: Replace eval with something that evaulates only math expressions for security reasons.
+
+    if (detailed) {
+        //Example, given expression was 3D6+5
+
+        return {
+            resultsPerRoll,    // e.g. [[4, 1, 1]] -- The three dice rolls of 3d6
+            rollDetailed: resolveRolls.join(' '), // e.g. (6) + 5  -- The resolved rolls
```
Comment (dimgrav) [inline src/lib/utils.js:52]: No single quotes! Mind the linter!

**Review by dimgrav — APPROVED** [review]
:beers:

**Pull request merged**